File: Find_Duplicates/Find_Duplicates.java
import java.util.HashSet;
public class Find_Duplicates {
    public static void main(String[] args) {
        int[] arr = {1, 2, 3, 2, 4, 5, 3};
        HashSet<Integer> seen = new HashSet<>();
        HashSet<Integer> duplicates = new HashSet<>();

        for (int num : arr) {
            if (!seen.add(num)) {
                duplicates.add(num);
            }
        }
        System.out.println("Duplicates: " + duplicates);
    }
}

File: Find_Duplicates/Find_Duplicates.py
def find_duplicates(arr):
    seen = set()
    duplicates = set()
    for num in arr:
        if num in seen:
            duplicates.add(num)
        else:
            seen.add(num)
    print("Duplicates:", list(duplicates))
arr = [1, 2, 3, 2, 4, 5, 3]
find_duplicates(arr)